Adds command+shift+a shortcut to instance a scene

likeich opened the pull request:
Adds command+shift+a as a default shortcut to instance a scene in the scene tree dock. This complements using command+a to add a node in the scene tree dock.

![Screenshot from 2021-03-31 20-35-41](https://user-images.githubusercontent.com/57882701/113227453-bb310600-9260-11eb-8a7d-01fab1835b88.png)

<!--
Pull requests should always be made for the `master` branch first, as that's
where development happens and the source of all future stable release branches.

Relevant fixes are cherry-picked for stable branches as needed.

Do not create a pull request for stable branches unless the change is already
available in the `master` branch and it cannot be easily cherry-picked.
Alternatively, if the change is only relevant for that branch (e.g. rendering
fixes for the 3.2 branch).
-->

KoBeWi commented:
Personally I use Ctrl + Shift + A, it's more consistent with Add Child Node.

LightningAA commented:
I prefer <kbd>Ctrl + Shift + A</kbd> too because it can be done with just the left hand (right hand doesn't have to come off mouse)

likeich committed:
Add ctrl+shift+a to instance scene in scenetree dock

Adds another key shortcut to instance a scene in the scentree dock. Complements ctrl+a to add a node.

KoBeWi commented:
The commits need to be squashed into one.

akien-mga commented:
Thanks!

akien-mga commented:
Cherry-picked for 3.4.

akien-mga commented:
Cherry-picked for 3.3.1.
